shared/gh/js/controllers/gh.api.user.js

```diff
@@ -69,7 +64,34 @@ define(['exports'], function(exports) {
                 return callback({'code': jqXHR.status, 'msg': jqXHR.responseText});
             }
         });
-        */
+        
+    };
+
+    /**
+     * Get the groups a user is a member of
+     *
+     * @param  {[type]}      userId               The ID of the user to get the group memberships for
+     * @param  {Function}    callback             Standard callback function
+     * @param  {Object}      callback.err         Error object containing the error code and error message
+     * @param  {Object}      callback.response    The groups the user is a member of
+     */
```

Comment:
I've been using `@throws` as well lately. Shall we add it wherever we thrown errors for parameter validation?